wallet/api/errors.go

```diff
@@ -156,6 +157,7 @@ var (
 	ErrUserRejectedAccessToKeys                           = errors.New("the user rejected the access to the keys")
 	ErrUserRejectedSendingOfTransaction                   = errors.New("the user rejected the sending of the transaction")
 	ErrUserRejectedSigningOfTransaction                   = errors.New("the user rejected the signing of the transaction")
+	ErrUserRejectedCheckingOfTransaction                  = errors.New("the user rejected the checking of the transaction")
```

Comment:
```suggestion
	ErrUserRejectedCheckingOfTransaction                  = errors.New("the user refused to check the transaction")
```

I realise rewording this means the 2 above follow a different pattern

Reply:
This is fixed now. Thanks!